taker-frontend/src/components/LiquidationPrice.ts

```diff
@@ -14,3 +14,16 @@ export function liquidationPriceQuantoContracts(leverage: number, quantity: numb
         return price + quantoLiquidationPrice(leverage, quantity, price);
     }
 }
+
+// returns the liquidation price for inverse contracts e.g. BTCUSD rounded to 0
+export function liquidationPriceInverseContracts(leverage: number, price: number, isLong: boolean) {
+    if (isLong) {
+        return Math.round(price * leverage / (leverage + 1));
+    } else {
+        if (leverage === 1) {
+            // With a leverage of 1 there is theoretically no liquidation
+            return 21_000_000;
```

Comment:
❓ This confused me in the `daemon` too. Why this number? The price can be `2^20-1` according to Olivia.

Reply:
It's an easter egg 😅